api/settings/base.py

```diff
@@ -60,6 +60,13 @@
 
 del TEMPLATE_CONTEXT_PROCESSORS[TEMPLATE_CONTEXT_PROCESSORS.index('apps.feedback.context_processors.feedback')]
 
+# Options for Premailer, which inlines the CSS on the fly in email templates and
+# makes all URLs absolute
+PREMAILER_OPTIONS = {"base_url": os.environ.get("PREMAILER_BASE_URL", "https://www.makeaplea.service.gov.uk"),
```

Comment:
It looks like the `https://www.makeaplea.service.gov.uk` is being redirected to `https://www.gov.uk/make-a-plea`. Do you think it may be better to use the latter?

** Same for: https://github.com/ministryofjustice/manchester_traffic_offences_pleas/pull/321/files#diff-6b1f5516f9f8930af4e1b2a32aea9b43R113